recipe/meta.yaml

```diff
@@ -73,9 +73,6 @@ test:
     # Need these for AOT. Do not init msvc as it may not be present
     - {{ compiler('c') }}      # [not win]
     - {{ compiler('cxx') }}    # [not win]
-    # Working around an issue with NumPy 1.19
-    # xref: https://github.com/numba/numba/issues/5929
-    - numpy >=1.15,<1.19
```

Comment:
Oh sorry spoke too soon. Yeah let's bump the build number if we are changing this

Reply:
This is only a test constraint. Shouldn't affect users.

Reply:
I though it was an install constraint, which would need a build number bump

Reply:
Oops 🤦‍♂️ Missed your second comment. Agree we don't need to bump the build number

Reply:
We went through the exact same sequence. :)